mlrun/api/db/sqldb/models/models_mysql.py

```diff
@@ -213,6 +213,28 @@ class Run(Base, HasStruct):
         def get_identifier_string(self) -> str:
             return f"{self.project}/{self.uid}/{self.iteration}"
 
+    class BackgroundTask(Base, BaseModel):
+        __tablename__ = "background_task"
+        __table_args__ = (
+            UniqueConstraint("name", "project", name="_background_task_uc"),
+        )
+
+        id = Column(Integer, primary_key=True)
+        name = Column(
+            String(255, collation=SQLCollationUtil.collation()), nullable=False
+        )
+        project = Column(String(255, collation=SQLCollationUtil.collation()))
```

Comment:
Is the default for nullable is `True` ? is it ok to have a unique constraint on a nullable field ?

Reply:
he default for nullable is `True`. Added default name for none-project background task.

Reply:
Why we want project to be nullable ?

Reply:
we don't changed.